client/components/chart/style.scss

```diff
@@ -434,18 +434,11 @@ $y-axis-padding: 0 20px 0 10px;
 			min-width: 0;
 
 			&::before {
-				content: '';
+				@include long-content-fade( $color: var( --color-primary-rgb ) );
```

Comment:
What if we use here the same color used for the [background](https://github.com/Automattic/wp-calypso/blob/fea949a8e688300e1dde2b51b8dbb451bf1721f4/client/components/chart/style.scss#L111)?

```suggestion
				@include long-content-fade( $color: var( --color-neutral-600-rgb ) );
```

That way it'll fade to the background of the tooltip, which looks a bit nicer IMO:

<img width="288" alt="Screen Shot 2019-09-13 at 12 33 31" src="https://user-images.githubusercontent.com/1233880/64879046-ff2ecc80-d622-11e9-9e8f-760d25eaf3c3.png">

Reply:
I prefer this approach too

Reply:
<img width="392" alt="Screen Shot 2019-09-13 at 2 10 19 PM" src="https://user-images.githubusercontent.com/2810519/64884658-41aad600-d630-11e9-989a-5d770594f127.png">

Also looks much, much better when the post title does not overflow 👍